src/retrace-server-reposync

```diff
@@ -207,11 +208,11 @@ if __name__ == "__main__":
 
     if CONFIG["UseFafPackages"]:
         log_info("Creating repo from faf")
-        targetid = "%s-%s-%s" % (distribution, version, arch)
-        targetdir = os.path.join(CONFIG["RepoDir"], targetid)
+        targetid = Path("%s-%s-%s" % (distribution, version, arch))
```

Comment:
wat, this isn’t even supposed to be a path.

Reply:
Oops, sorry. Fixed in a new commit to this branch.